airbyte-server/src/main/java/io/airbyte/server/converters/SchemaConverter.java

```diff
@@ -24,107 +24,87 @@
-    final List<Stream> persistenceStreams =
-        sourceSchema.getStreams()
-            .stream()
-            .map(apiStream -> {
-              final List<Field> persistenceFields = apiStream.getFields()
-                  .stream()
-                  .map(SchemaConverter::toPersistenceField)
-                  .collect(Collectors.toList());
+  public static io.airbyte.api.model.AirbyteStream convertTo(final io.airbyte.protocol.models.AirbyteStream stream) {
+    return new io.airbyte.api.model.AirbyteStream()
+        .name(stream.getName())
+        .jsonSchema(stream.getJsonSchema())
+        .supportedSyncModes(Enums.convertListTo(stream.getSupportedSyncModes(), io.airbyte.api.model.SyncMode.class))
+        .sourceDefinedCursor(stream.getSourceDefinedCursor())
+        .defaultCursorField(stream.getDefaultCursorField());
+  }
 
-              return new Stream()
-                  // immutable
-                  // todo (cgardens) - not great that we just trust the API to not mutate these.
-                  .withName(apiStream.getName())
-                  .withFields(persistenceFields)
-                  .withSupportedSyncModes(apiStream.getSupportedSyncModes()
-                      .stream()
-                      .map(e -> Enums.convertTo(e, io.airbyte.config.SyncMode.class))
-                      .collect(Collectors.toList()))
-                  .withSourceDefinedCursor(apiStream.getSourceDefinedCursor())
-                  .withDefaultCursorField(apiStream.getDefaultCursorField())
-                  // configurable
-                  .withSyncMode(Enums.convertTo(apiStream.getSyncMode(), io.airbyte.config.SyncMode.class))
-                  .withCursorField(apiStream.getCursorField())
-                  .withSelected(persistenceFields.stream().anyMatch(Field::getSelected));
-            })
-            .collect(Collectors.toList());
+  public static io.airbyte.protocol.models.AirbyteStream convertTo(final io.airbyte.api.model.AirbyteStream stream) {
+    return new io.airbyte.protocol.models.AirbyteStream()
+        .withName(stream.getName())
+        .withJsonSchema(stream.getJsonSchema())
+        .withSupportedSyncModes(Enums.convertListTo(stream.getSupportedSyncModes(), io.airbyte.protocol.models.SyncMode.class))
+        .withSourceDefinedCursor(stream.getSourceDefinedCursor())
+        .withDefaultCursorField(stream.getDefaultCursorField());
+  }
 
-    return new Schema().withStreams(persistenceStreams);
+  public static io.airbyte.api.model.AirbyteCatalog convertTo(final io.airbyte.protocol.models.AirbyteCatalog catalog) {
+    return new io.airbyte.api.model.AirbyteCatalog()
+        .streams(catalog.getStreams()
+            .stream()
+            .map(SchemaConverter::convertTo)
+            .map(s -> new io.airbyte.api.model.AirbyteStreamAndConfiguration()
+                .stream(s)
+                ._configuration(getDefaultAirbyteStreamConfiguration(s)))
```

Comment:
why is this underscored?

Reply:
This is how the code generator spat out, maybe because it's some sort of keyword?

I can change the name to avoid this

Reply:
nah. it's okay. as long as it is an intentional thing to avoid a namespace confusion i think it's okay.